### PR TITLE
Only use commit statuses that start with 'ci/' in ci-status check

### PR DESCRIPTION
**Pull request by bss**

### CODE REVIEW

File: lib/hub/commands.rb [hunk 1]
```diff
@@ -90,8 +90,9 @@ def ci_status(args)
         abort "Aborted: no revision could be determined from '#{ref}'"
       end
 
-      statuses = api_client.statuses(project, sha)
-      status = statuses.first
+      all_statuses = api_client.statuses(project, sha)
+      ci_statuses = all_statuses.select { |s| s['context'].start_with? 'ci/' }
+      status = ci_statuses.first
```
Comment (nikhilmat) [inline lib/hub/commands.rb]: can use `all_statuses.find`

**Comment by nikhilmat** [comment]
Looks good, just have to get everyone to update their machines now

@alfred accept

**Comment by zp-alfred** [comment]
Accepted by nikhilmat!

**Comment by nikhilmat** [comment]
The similar change needs to be made in sensei for the deploy blockers stuff

**Comment by bss** [comment]
@nikhilmat Matt W already did that here https://github.com/Gusto/sensei/pull/2

**Pull request closed without merging**